languages/csharp-dotnet-core2/template/src/Algorithm.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using Algorithmia;
+
+//This is an example of an algorithm. You can test it by copy/pasting the following into the web editor:
+/// {"name": "Algorithmia user"}
+///
+/// You should expect to see the following as output:
+/// {"output":"Hello Algorithmia user"}
+
+/// 'Algo' is the primary namespace of your algorithm.
+namespace Algo
+{
+    /// The input object type for your algorithm, the types defined here must be json serializable via the 'Newtonsoft.Json' package.
```

Comment:
I don't think we want to bother the normal user by talking about the fact that it `must be json serializable via the 'Newtonsoft.Json'`. Even with my C# knowledge, I'm not sure what that means. Maybe say all methods should be public?

Reply:
I guess you're right to some degree, I just don't want them to get confused when something doesn't parse right.